modules/imaging_browser/php/Imaging_Session_ControlPanel.class.inc

```diff
@@ -96,13 +96,17 @@ class Imaging_Session_ControlPanel
             $params['PCandID'] = $timePoint->getCandID();
             $params['PVL']     = $timePoint->getVisitLabel();
         }
-        $tarchiveIDs = $DB->pselect(
-            "SELECT TarchiveID 
-            FROM tarchive 
+
+        // To enable DICOM download: Add pair: tarchiveID, ArchiveLocation
+        $qresult = $DB->pselectRow(
+            "SELECT TarchiveID, ArchiveLocation
+            FROM tarchive
             WHERE PatientName LIKE $ID",
```

Comment:
@christinerogers 

discussed at the Imaging meeting of July 18th, 2017:
1) pselectRow should be pselect (we might have more than one tarchiveID per session)
`$new_array = array();
foreach($qresult as $k=>$v) {

    $new_array['TarchiveID'][$k] = $v['TarchiveID'];
    $new_array['ArchiveLocation'][$k] = $v['ArchiveLocation'];
}
`
However, the object `$subject.tarchiveids` and `$subject.tarchiveidLoc` in the template needs to handle the tarchiveID and ArvhiveLocation simultaneously (somehow)

2) Have the download DICOM button next to the Dicom Archive Link; and /or have the download cloud list the Dicom Tarchive ID next to it

3) add the download cloud in the dicom archive module as well (RM #9865); @davidblader